nemo/collections/asr/models/enhancement_models.py

```diff
@@ -434,6 +434,10 @@ def evaluation_step(self, batch, batch_idx, dataloader_idx: int = 0, tag: str =
         # Log global step
         self.log('global_step', torch.tensor(self.trainer.global_step, dtype=torch.float32), sync_dist=True)
 
+        if type(self.trainer.val_dataloaders) == list and len(self.trainer.val_dataloaders) > 1:
```

Comment:
Can you do isinstance(..., (list, tuple)) and ...

Reply:
Yes sure.